cli/cmd/stat.go

```diff
@@ -389,3 +411,53 @@ func sortStatsKeys(stats map[string]*row) []string {
 	sort.Strings(sortedKeys)
 	return sortedKeys
 }
+
+// Perform all validation on the command-line options, returning the first
+// error encountered, or `nil` if the options are valid.
+func (o *statOptions) validate(resourceType string) error {
```

Comment:
This is minor, but standard godoc convention is to start all function comments with the function name itself. From [golang.org](https://blog.golang.org/godoc-documenting-go-code):

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Mind rewriting all of the function comments that you're adding in this diff to comply?